.eslintrc.json

```diff
@@ -19,6 +19,8 @@
         "newlines-between": "always",
         "alphabetize": { "order": "asc" }
       }
-    ]
+    ],
+    "jsx-a11y/no-static-element-interactions": "off",
+    "jsx-a11y/click-events-have-key-events": "off"
```

Comment:
Both of these can be enabled once we convert all the `div` items with an `onClick` to be html `button` elements. I tried it but it's having a visual impact in a few cases - we'll need to revisit carefully in the future...better to enable the larger rulesets in favor of progress.